code/modules/clothing/masks/monitor.dm

```diff
@@ -45,8 +45,8 @@
 		)
 
 /obj/item/clothing/mask/monitor/set_dir()
-	dir = SOUTH
```

Comment:
This is deliberate, it has null states for some directions and will be invisible if dragged/discarded while not equipped.

Reply:
`dir` should never be changed outside of the `set_dir()`-proc and even VV is re-routed through it, so with this override as long as `dir` was initially set to `SOUTH` it will remain so.

Anything that directly sets `dir` should be replaced with a `set_dir()`-call, due to event handling if nothing else.

Reply:
I see now that there are some cases where `dir` is manually set, bypassing `set_dir()`, looking rather deliberate. I'll arrange something.

Reply:
I feel it's fine to not call parent in this proc specifically if the dir is left changed, or to modify what the given arguments are before calling parent. In this example, for instance, you could instead do `..(SOUTH)`.

Reply:
The problem, as I see it, there are pieces of code doing `dir = DIRECTION` and overriding `set_dir()` to prevent others from changing said direction.
I've got a solution for it.

Reply:
Solution implemented for the override I noted was bypassed via `dir = direction`